storage/src/main/java/org/apache/kafka/server/log/remote/metadata/storage/ConsumerTask.java

```diff
@@ -157,9 +157,10 @@ private void processConsumerRecord(ConsumerRecord<byte[], byte[]> record) {
             remotePartitionMetadataEventHandler.handleRemoteLogMetadata(remoteLogMetadata);
             readOffsetsByUserTopicPartition.put(remoteLogMetadata.topicIdPartition(), record.offset());
         } else {
-            log.debug("The event {} is skipped because it is either already processed or not assigned to this consumer", remoteLogMetadata);
+            log.trace("The event {} is skipped because it is either already processed or not assigned to this consumer",
```

Comment:
@abhijeetk88 

This log statement will be chatty and prints the entire event. Reduced the logger level from `debug` to `trace`.